main.go

```diff
@@ -45,53 +48,21 @@ func main() {
 
 	// get base64 encoded credentials argument from action input
 	credentialsStr := githubactions.GetInput(credentialsInput)
-	if credentialsStr == "" {
-		missingInput(credentialsInput)
-	}
 	// add base64 encoded credentials argument to mask
 	githubactions.AddMask(credentialsStr)
```

Comment:
Does this have any issues if `credentialsStr` is blank? Previously the function would abort before reaching this.

Reply:
A warning will be displayed in the action run but the run will carry on. 
![image](https://github.com/SyntyStudios/gdrive-upload-action/assets/37458498/bfd73468-7e68-4f4f-a515-5e8c5bfa10ca)